community/server/src/docs/ops/security.asciidoc

```diff
@@ -129,8 +126,9 @@ This provides a number of advantages:
   Allow from 192.168.0
 </Proxy>
 ---------------
-
-While equivalent functionality can be implemented with Neo4j's +SecurityRule+ plugins (see above), for operations professionals configuring servers like Apache is often preferable to developing plugins. However it should be noted that where both approaches are used, they will work harmoniously providing the behavior is consistent across proxy server and +SecurityRule+ plugins.
++
+While equivalent functionality can be implemented with Neo4j's +SecurityRule+ plugins (see above), for operations professionals configuring servers like Apache is often preferable to developing plugins.
```

Comment:
This was a strange sentence. I suggest:

While it is possible to develop plugins using Neo4j's +SecurityRule+ (see above), operations professionals would often prefer to configure servers such as Apache.

I still find this is a bit suggestive about what operations professionals are supposed to do, but I'm not sure we want to use the word 'recommend' here, although we could.